chai-as-promised/.editorconfig

```diff
@@ -0,0 +1,7 @@
+root = true
```

Comment:
@cliffburger I [created an issue](https://github.com/borisyankov/DefinitelyTyped/issues/4211) concerning the EditorConfig files.